js/src/lib/components/InferenceWidget/widgets/TextGenerationWidget/TextGenerationWidget.svelte

```diff
@@ -61,7 +61,9 @@
 		if (exampleOutput) {
 			output = exampleOutput.text;
 			outputJson = "";
-			renderTypingEffect(output);
+			// if output doesn't start with space, add space in front of output
+			const prefix = /^\s/.test(output) ? "" : " ";
+			renderTypingEffect(prefix+output);
```

Comment:
while working [here](https://huggingface.co/01-ai/Yi-34B/discussions/17), I've discovered a subtle UX bug.

The bug is this:
Unlike other widgets that show `output.text` in seperate box, TextGen widget shows the output as a continuation of the input text area (see attached img). Therefore, for TextGen widget, user is expected to supply `output.text` that starts with a space. This PR fixes this bug

<img src="https://github.com/huggingface/hub-docs/assets/11827707/9da079f8-a8ac-4179-b8b2-24552eb70f83" width="500">